backend/test/fixtures/users.yml

```diff
@@ -59,3 +59,12 @@ administrator:
     - api_entreprise:reporter
     - franceconnect:reporter
     - api_particulier:reporter
+
+<% 1.upto(50) do |i| %>
+user_test_<%= i %>:
+  email: "user_test_<%= i %>@yopmail.com"
+  roles:
+    - api_entreprise:instructor
```

Comment:
Yes I was using them to see how a big UserList Table would behave. Do you want me to delete it now ?

Reply:
my point is that these users are instructors, instead of being simple user wich is confusing regarding the name.

I suggest you keep the users but remove their roles.